scripts/create_tables.sql

```diff
@@ -1 +1,29 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors(
+  id SERIAL PRIMARY KEY,
+  surname TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on INTEGER,
+  specialty TEXT
+);
+
+CREATE TABLE patients(
+  id SERIAL PRIMARY KEY,
+  surname TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on INTEGER,
+  insurance TEXT
+);
+
+CREATE TABLE appointments(
+  id SERIAL PRIMARY KEY,
+  date_time TIMESTAMP,
+  comments TEXT,
+  patient_id INTEGER REFERENCES patients(id),
+  doctor_id INTEGER REFERENCES doctors(id)
+);
+
+CREATE INDEX ON appointments(patient_id);
+CREATE INDEX ON appointments(doctor_id);
```

Comment:
is `born_on` really an integer? how would you convert an integer to a birth date?